app/controllers/oauth_clients_controller.rb

```diff
@@ -65,6 +65,14 @@ def callback
 
   private
 
+  def set_oauth_state
+    @oauth_state = params[:state]
+  end
+
+  def clear_oauth_state_cookie
+    cookies.delete "oauth_state_#{@oauth_state}" unless @oauth_state.nil?
```

Comment:
Code readability ...

```suggestion
    cookies.delete("oauth_state_#{@oauth_state}") unless @oauth_state.nil?
```